src/main.py

```diff
@@ -130,13 +130,15 @@ def order_subscription():
 @login_required
 def order_packets():
     form = OrderPacketsForm()
+    if current_user.user_type == manifest.USER_TYPES.ADMIN:
+        form.email.choices = [(email, email) for email in db.get_all_emails()]
     descriptions = [(manifest.PACKETS[name]["name"], manifest.PACKETS[name]["description"]) for name in
                     manifest.PACKETS]
     descriptions = dict(descriptions)
 
     if form.validate_on_submit():
-        user = db.get_client_by_email(form.email.data)
-        user.bundles.append(BundleInfo(email=form.email.data,
+        user = db.get_client_by_email(form.email.data if form.email.data != "current_user" else current_user.email)
+        user.bundles.append(BundleInfo(email=user.basic.email,
```

Comment:
Minor inconvenience.  It is possible to have copies of the same bundles. User can buy multiple times the same thing, and all of it will be listed in user profile. Shouldn't we check if the bundle is already bought?

Reply:
Should be covered by SCRUM-53